src/components/donation/Steps/Submit/Crypto/Checkout.tsx

```diff
@@ -0,0 +1,113 @@
+import Image from "components/Image/Image";
+import { Info, LoadingStatus } from "components/Status";
+import { chains } from "constants/chains";
+import { isDisconnected, useWalletContext } from "contexts/WalletContext";
+import { maskAddress } from "helpers";
+import { PropsWithChildren } from "react";
+import { CryptoSubmitStep } from "slices/donation";
+import { ConnectedWallet } from "types/wallet";
+import TxSubmit from "./TxSubmit";
+import WalletSelection from "./WalletSelection";
+
+type Props = CryptoSubmitStep & { classes?: string };
+
+export default function Checkout({ classes = "", ...props }: Props) {
```

Comment:
this component is previously `<Crypto/>` but renamed to `<Checkout/>` as  gas fee (obtained via simulation which needs wallet) is now not included in donation summary items